### DISCUSSION TITLE
Import/export KeePassXC app settings?

### DISCUSSION THREAD

**Discussion by bzgec** [keepassxreboot/keepassxc#7857] [Q&A] [answered]
Is it possible to import/export KeePassXC app settings (like for KeePassXC Browser Extension)?

**Comment by droidmonkey** [comment, marked as answer]
You can do both direct in the browser extension settings. For the main app you need to pull the .ini files from their respective folders.

**Reply by bzgec** [reply]
Thank you! :)

For others looking for the `.ini` flie: under Linux look at `~/.config/keepassxc/`.

**Reply by syakovyn** [reply]
What about OSX? There is no ~/.config/keepassxc/ and I can't find where the app settings are stored.

**Reply by syakovyn** [reply]
Found. `~/Library/Application Support/KeePassXC` and `~/Library/Caches/KeePassXC`

**Reply by DeividasJackus** [reply]
In case this Windows users stumble here: look for `%APPDATA%\KeePassXC\keepassxc.ini` as per https://github.com/keepassxreboot/keepassxc/issues/2574
